src/main.rs

```diff
@@ -12,7 +12,7 @@ async fn main() {
 
     // Start server
     let port = std::env::var("SERVER_LOCAL_PORT").unwrap_or("3000".to_owned());
-    let addr: SocketAddr = format!("127.0.0.1:{port}").parse().unwrap();
+    let addr: SocketAddr = format!("0.0.0.0:{port}").parse().unwrap();
```

Comment:
Why was the address changed here?

Reply:
I am removing the map from localhost to all addresses for now as localhost will not always be valid in all environments

Reply:
Ohhh okay

Reply:
Could you use a env variable too for this one and default the value to 0.0.0.0?

Reply:
okay